cypress/e2e/accessibility.cy.ts

```diff
@@ -42,30 +38,30 @@ describe("Accessibility tests in dark mode", () => {
 
     it("Checks clients page", () => {
         cy.visit("/clients");
-        cy.get(".dark-button").click();
+        cy.get("label[aria-label='Theme Switch']").click();
 
         cy.checkAccessibility();
     });
 
     it("Checks lists page", () => {
         cy.visit("/lists");
-        cy.get(".dark-button").click();
+        cy.get("label[aria-label='Theme Switch']").click();
 
         cy.checkAccessibility();
     });
 
     it("Checks calendar page", () => {
         cy.visit("/calendar");
-        cy.get(".dark-button").click();
-
         cy.get("table").should("be.visible");
+        cy.get("label[aria-label='Theme Switch']").click();
+
         cy.checkAccessibility();
     });
 
     it("Checks login page", () => {
         cy.visit("/clients");
-        cy.get(".dark-button").click();
-        cy.get("input[type=submit]").contains("Sign out").click();
+        cy.get("label[aria-label='Theme Switch']").click();
```

Comment:
What is the benefit of targeting using label aria label instead of input type? Just curious.

Reply:
Input type is checkbox according to MUI, was worried if people use checkboxes elsewhere it might pick up on those as well and give error of multiple elements exist.